index.html

```diff
@@ -23,18 +34,34 @@
         <time></time>
       </div>
     </header>
+    <footer class="description">
+    </footer>
+
+          <a href="#" class="readon"></a>
   </article>
   </section>
 
+<aside id="myinfo">
   <address>
+    <ul><h4>My Interests Are:</h4>
+      <li>Music</li>
+      <li>Coding</li>
+      <li>X-files</li>
+      <li>Calvin and Hobbes Comic</li>
+      <li>...and MANY more!!</li>
+      <li>E-mail me below to find out more.</li>
+    </ul>
   My E-mail address is 'dzknj@outlook.com'<br>
     <a href="mailto:dzknj@outlook.com">Click Here to E-mail me</a>
   </address>
-
+</aside>
   <script src="jquery-1.12.0.js"></script>
+  <script src="js/projectview.js"></script>
   <script src="js/data.js"></script>
   <script src="js/app.js"></script>
 
 </body>
-<div class="footer"><div>
+<div class="footer">
+<h1>Thank you for Visiting!!</h1>
```

Comment:
I'd argue making this an h3 and swapping with your h3 at the top so that one is the h1. Also indent this inside the div.